guides/common/modules/proc_setting-up-flatpak-remote-for-smartproxy.adoc

```diff
@@ -20,9 +20,15 @@ For more information, see xref:Synchronizing_Repositories_{context}[].
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-$ flatpak remote-add --authenticator-name=org.flatpak.Authenticator.Oci katello oci+https://{smartproxy-example-com}/pulpcore_registry/
-----
-. Log in using Podman:
+$ flatpak remote-add --authenticator-name=org.flatpak.Authenticator.Oci katello oci+https://{smartproxy-example-com}/
+----
+. Log in the host to the container registry using one of the following methods: 
+* Certificate authentication
+** When registering a host to {SmartProxy}, select the *Set up container registry certs* checkbox.
+** If the host is already registered, run the *Flatpak - Login to registry via podman* job template on the host.
+** Set *Set up certificate authentication* to `true` and enter the URL of the {SmartProxy} server as the registry URL.
```

Comment:
Same proposal as above (merge in one step?).

One more thing - we may need to update the _note_ about global availability of Flatpaks on Capsule (it should be scoped by org/lce/cv in 6.18). But that may go to another PR later.

Reply:
Same as above.